Cargo.toml

```diff
@@ -80,7 +80,11 @@ comfy-table = { version = "6.1", default-features = false }
 const_panic = { version = "0.2.8", features = ["rust_1_64"] }
 crossbeam = "0.8"
 ecolor = "0.22.0"
-eframe = { version = "0.22.0", default-features = false, features = ["x11"] }
+eframe = { version = "0.22.0", default-features = false, features = [
+  "default_fonts",
+  "wayland",
+  "x11",
+] }
```

Comment:
There are also features set in `crates/re_viewer/Cargo.toml` - let's pick one or the other for setting feature flags 😬

Reply:
Hmm they are different concerns though: the features in the root manifest here are the required features that all crates in the workspace _must_ enable whenever they try and use `eframe`... so I think at least `x11` & `wayland` should be here no matter what (don't know about defaut_fonts though?).